src/main/java/org/sagebionetworks/web/client/widget/accessrequirements/createaccessrequirement/CreateManagedACTAccessRequirementStep2.java

```diff
@@ -101,6 +101,7 @@ public void configure(ManagedACTAccessRequirement accessRequirement) {
 		view.setIsIDUPublic(accessRequirement.getIsIDUPublic());
```

Comment:
ACT UI for creating a Managed AR.  Initialize to AR config, and update from view on save.